main.go

```diff
@@ -1,16 +1,36 @@
 package main
 
 import (
-	"io"
-	"log"
+	"fmt"
 	"net/http"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+	http_router "github.com/julienschmidt/httprouter"
```

Comment:
Also, take a look at https://godoc.org/github.com/go-chi/chi

Reply:
Sure, I'll look into the router package you requested